com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/persistence/service/ConfigurationEntryService.java

```diff
@@ -22,15 +20,13 @@
 import com.sap.cloud.lm.sl.common.util.JsonUtil;
 import com.sap.cloud.lm.sl.mta.model.Version;
 
-@Named
 public class ConfigurationEntryService extends PersistenceService<ConfigurationEntry, ConfigurationEntryDto, Long> {
 
-    @Inject
     protected ConfigurationEntryMapper entryMapper;
 
-    @Inject
-    public ConfigurationEntryService(EntityManagerFactory entityManagerFactory) {
+    public ConfigurationEntryService(EntityManagerFactory entityManagerFactory, ConfigurationEntryMapper entryMapper) {
```

Comment:
Can we avoid the complexity of having this object be built with Spring bean factories by keeping the `@Named` and `@Inject` annotations and add a `@Primary` annotation to the `CredstoreConfigurationEntryMapper` class from the internal project? `@Profile` annotations might also be an option if the first doesn't work.